src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP4200/CreateTransactionEOF1Filler.yml

```diff
@@ -0,0 +1,419 @@
+---
+CreateTransactionEOF1:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      # Valid EOF deployed code with RJUMP having JUMPDEST as target
+      # - Positive:
+      # Deployed code: RJUMP(3) STOP RETURN STOP JUMPDEST PUSH1(1) PUSH1(1) SSTORE STOP - ef000101000d005c000300f3005b600160015500
```

Comment:
To avoid unreachable code this could be something like
`RJUMP(3) SSTORE STOP JUMPDEST PUSH1(1) PUSH1(1) RJUMP(-10)`

Reply:
Maybe I would just delete all the tests with JUMPDEST, it is now NOP instruction, not better than any other instruction...

Reply:
Also I wan to remove combinations that test whether it's validated in deployed code.

I think it's enough to try only initcode, doesn't seem worth it to check all tiny details of validation in both.